aws/eks-helm/jenkins/README.md

```diff
@@ -0,0 +1,78 @@
+<!-- BEGIN_TF_DOCS -->
+## Requirements
```

Comment:
Please define the requirement

Reply:
I have updated the PR with what you suggested